src/tracer.c

```diff
@@ -40,21 +41,56 @@ tracee_main(char **argv)
 	FATAL_ERROR; // exec shouldn't return.
 }
 
-int
-is_open_syscall(int syscall)
+/*
+    addr is an address pointing to the tracee process' address space, thus we need to copy it.
+*/
+char *
+read_path(char *addr, pid_t pid)
```

Comment:
something like `read_str_from_process` perhaps?

Reply:
the process needs to be a tracee, maybe `read_str_from_tracee_process`? Or is it too verbose?